contentcuration/contentcuration/assets/channelList/router.js

```diff
@@ -0,0 +1,46 @@
+// include all logic in "base" entrypoint
+import VueRouter from 'vue-router';
+import ChannelList from './views/Channel/ChannelList';
+import ChannelModal from './views/Channel/ChannelModal';
+import ChannelSetList from './views/ChannelSet/ChannelSetList';
+import ChannelSetModal from './views/ChannelSet/ChannelSetModal';
+import { ListTypes, RouterNames } from './constants';
+
+const router = new VueRouter({
+  routes: [
+    {
+      name: RouterNames.CHANNELS,
+      path: '/channels/:listType',
+      props: true,
```

Comment:
Yes - previously there were four routes defined, with the prop being explicitly passed, rather than the value being passed straight through from the URL param.